ZahraArshia/book-flight-frontend
# Feature/reservations page

## Changes and Features:
- reservations page gets and displays data correctly
- move the function that is responsible for navigation inside the homepage file
- fix a typo in the Flights component inside the homepage file

## How to test:
Rails API:
- run `rails c` in the terminal of our rails API.
- inside the `rails c` box, create some reservations (you will need to create two users at least and some items to do that)
- run `rails s -p 3001` in the terminal of the rails API.

React app:
- log in with the user that has the reservations that you created.
- you should see these reservations under the directory of `http://127.0.0.1:3000/reservations`

***Note:*** *When I checked the reservations. I noticed that the main page displays the flights only if I removed the **Slider tag** that wraps the flights component. Please @ZahraArshia. Check that later when you have time.*    

***Note:*** *This [PR](https://github.com/ZahraArshia/book-flight-backend/pull/24) needs to get approved before testing this PR.*
